src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -379,16 +379,18 @@ protected function throttle(Throwable $e)
     /**
      * Remove the given exception class from the list of exceptions that should be ignored.
      *
-     * @param  string  $exception
+     * @param  string|array  $exceptions
      * @return $this
      */
-    public function stopIgnoring(string $exception)
+    public function stopIgnoring($exceptions)
```

Comment:
Can we use here array|string union ?

Reply:
@akiyamaSM we're not extensively using the union syntax right now in the framework so this isn't needed, thanks.

Reply:
Good to know @driesvints, may I know why?

Reply:
Just because we don’t want to do things in multiple ways.